lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java

```diff
@@ -102,6 +102,25 @@ public void setDelegate(@Nullable ImageAssetDelegate assetDelegate) {
       return putBitmap(id, bitmap);
     }
 
+    //if no images folder given assume drawable res
+    if (TextUtils.isEmpty(imagesFolder)) {
+      try
+      {
+        int drawableId = context.getResources().getIdentifier(filename, "drawable", context.getPackageName())
+        if(drawableId != -1)
+        {
+          //we dont pass "opts" we rely on system scaling drawables
```

Comment:
`// We don't pass BitmapFactor.Options because we rely on system scaling drawables.`

Reply:
fixed